wiki/Community/osu!_community_meetings/de.md

```diff
@@ -0,0 +1,43 @@
+
+Das nächste osu! Community Meeting findet voraussichtlich am: **22. Januar 2022 (Samstag 21:00 UTC)** statt.
+
+## Sich beteiligen
+
+Die osu! Community Meetings werden derzeit auf einer Discord Stage im [osu!dev Discord](https://discord.gg/ppy) gehostet und auf [peppy's Twitch Kanal](https://www.twitch.tv/ppy) zeitgleich ausgestrahlt. Jedes Treffen dauert voraussichtlich etwa eine Stunde und findet jedes zweite Wochenende statt.  
+
+Jeder ist herzlich eingeladen, mitzumachen! Wenn du eine Frage hast oder etwas zur Diskussion beitragen möchtest, kannst du auf der Discord Stage die Hand heben, um Dich in die Warteschlange einzureihen, oder eine Nachricht in den entsprechenden Discord-Thread schreiben, wenn Du Dich nicht traust zu sprechen, die dann laut vorgelesen und beantwortet wird. Bevor du eine Frage stellst, solltest du außerdem die folgenden Richtlinien beachten:
+
+- Plane deine Gesprächsinhalte im Voraus und halte sie so kurz wie möglich.
+- Versuche nur Themen anzusprechen, die die gesamte Community betreffen, nicht nur dich.
+- Eine Verifizierung auf osu!dev ist nicht notwendig, kann aber deine Chancen erhöhen, für Fragen während der Q&A-Sessions ausgewählt zu werden.
+
+Wir bemühen uns, nicht mehr als 5 Minuten pro Frage zu benötigen. Bitte habe daher Verständnis dafür, dass Fragen, die bereits beantwortet wurden oder besonders allgemein gehalten sind, möglicherweise nicht zur Diskussion gestellt werden.
+
+## Archiv
+
+Das erste osu! Community Meeting fand am 19. September 2021 statt. Alle Treffen wurden und werden auch künftig auf YouTube aufgezeichnet, zusammen mit einem begleitenden zusammenfassenden Dokument.
+
+| Meeting # | Link | Zusammenfassung | Agenda |
+| :-: | :-- | :-- | :-- |
+| 0 | [19. September 2021](https://youtu.be/-LjNll8VYTM) | [Sitzungsnotizen](https://gist.github.com/peppy/08b38cbac56a8b1e88133853a6a8d2fb) | Einführung in diese Meetings, Ranking-Kriterien, Schwierigkeitsgrade und Performance-Punkte, osu!lazer im Allgemeinen |
+| 1 | [3. Oktober 2021](https://youtu.be/D3OZCm6szVE) | [Sitzungsnotizen](https://docs.google.com/document/d/1-miPFZySYlYbtPW4G0828taycO1gh7dZ4icfSJCr4uA) | Diverse Fragen aus der Community, Entwicklungs-Update |
+| 2 | [17. Oktober 2021](https://youtu.be/nLT-fLeogrw) | [Sitzungsnotizen](https://docs.google.com/document/d/1wEQjzku_4IzvcWRanu0GkvWiPvb-IB7WzwU7Eyojj2E) | Verschiedene Fragen aus der Community |
+| 3 | [31. Oktober 2021](https://youtu.be/omvZbC7zJUI) | [Sitzungsnotizen](https://docs.google.com/document/d/1bhaMx--chdys8AbzSDOItbv710SVhsww2BYPzQg2rvw) | Verschiedene Fragen aus der Community |
+| 4 | [14. November 2021](https://youtu.be/rEWgTY-L95Y) | [Sitzungsnotizen](https://docs.google.com/document/d/1rhrz0zaxQfVK1UnvrcEqj4UMagTD0bPNh8wby8I38IA) | Performance-Punkte-Entwicklungsprozess, verschiedene Fragen aus der Community, Entwicklungs-Update |
```

Comment:
```suggestion
| 4 | [14. November 2021](https://youtu.be/rEWgTY-L95Y) | [Sitzungsnotizen](https://docs.google.com/document/d/1rhrz0zaxQfVK1UnvrcEqj4UMagTD0bPNh8wby8I38IA) | Entwicklungsprozess der Performance-Punkte, verschiedene Fragen aus der Community, Entwicklungs-Update |
```
I'm not really a fan of cluttering words together like here. Though, I'm fine if you want to keep it like this.

Reply:
yeah, this is better tbh